docs/topics/coroutines-guide.md

```diff
@@ -31,6 +31,6 @@ In order to use coroutines as well as follow the examples in this guide, you nee
 
 ## Additional references
 
-* [Guide to UI programming with coroutines](https://github.com/Kotlin/kotlinx.coroutines/blob/master/ui/coroutines-guide-ui.md)
-* [Coroutines design document (KEEP)](https://github.com/Kotlin/kotlin-coroutines/blob/master/kotlin-coroutines-informal.md)
+* [Guide to UI programming with coroutines](https://github.com/Kotlin/kotlinx.coroutines/blob/develop/ui/coroutines-guide-ui.md)
```

Comment:
But it has to be pointing to the current `master` which is the state of the last stable release, not to develop.

Maybe the problem is in something elese?

Reply:
```suggestion
* [Guide to UI programming with coroutines](https://github.com/Kotlin/kotlinx.coroutines/blob/master/ui/coroutines-guide-ui.md)
```

Reply:
Oh, it's my bad. 
I was on the `develop` branch when preparing the PR and copy the wrong link.